server/apps/common/models/base_model.py

```diff
@@ -157,28 +157,31 @@ def update(self, filter, doc, etag=None):
         return res
 
     def replace(self, filter, doc, etag=None):
```

Comment:
Since we are changing this file already, should we also rename the `filter` parameter in a couple of methods, because it clashes with the name of a built-in class?